config.inc.php.template

```diff
@@ -18,9 +18,20 @@ define('OBJECTCLASS', 'inetOrgPerson');
 define('PAGETITLE', 'example.com User Admin');
 
 define('MAIL_SENDER', 'webteam@example.com');
-define('MAIL_SUBJECT', 'Your account for example.com');
```

Comment:
Line 204 in addUser.php still references MAIL_SUBJECT (I think that line can be deleted)